src/style.css

```diff
@@ -1 +1,92 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+}
+
+.search-bar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  margin-top: 20px;
+  box-sizing: border-box;
+}
+
+.search-bar__logo {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  padding-left: 26px;
+}
+
+.search-bar__logo--small {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  padding-left: 13px;
```

Comment:
Don't use tag names for styling (except 'html' and 'body'). It is recommended to use classes for styling. If '.search-bar__logo' and '.search-bar__logo--small' are meant to style 'img' tags, consider adding a class to the 'img' tags instead.